packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Provides themeable 10px on top and bottom
+   */
+  padVertical: PropTypes.bool,
```

Comment:
Would we want to update these prop names to `hasPaddingVertical` and `hasPaddingHorizontal` to reflect our boolean naming standard and to reflect the default value of true?